tests/nightly/unity/task.yaml

```diff
@@ -5,19 +5,14 @@ environment:
 
 systems: [ubuntu-16.04-64]
 
-execute:
-    # FIXME Test is broken. Downloads HUNDREDS and HUNDREDS of packages,
-    # and asks for input on configuration file diffs.
```

Comment:
The option `DEBIAN_FRONTEND=noninteractive` is for debconf based prompts only. Most configfile prompts are done via dpkg directly which does not involve debconf (the exception is ucf which can be used for conffile handling outside of the dpkg infrastructure and it uses debconf). To stop dpkg from prompting the option `--force-conf{old,new}` must be passed to dpkg (e.g. via apt environment). But like fgiminez said, we probably won't want to use it because we want to know if there are conffile prompts (at least currently).